test-tool/iscsi-test-cu.c

```diff
@@ -1404,19 +1405,6 @@ main(int argc, char *argv[])
                 }
         }
 
-        /* try reading block device characteristics vpd */
```

Comment:
Converting this code into a unit test seems wrong to me because several tests need the block device characteristics page contents (test-tool/test_sanitize_block_erase.c, test-tool/test_sanitize_crypto_erase.c and test-tool/test_sanitize_overwrite.c) and converting this code into a unit test creates a dependency between tests that is hard to track.

Reply:
I feel that such tests should first check if the BDC page is valid before running the test. Block Device Characteristics page is not mandatory and when checking for compliance, it should not be a mandatory test for the compliance suite to work.